File: src/main/java/org/tdl/vireo/model/repo/DepositLocationRepo.java
package org.tdl.vireo.model.repo;

import org.tdl.vireo.model.DepositLocation;
import org.tdl.vireo.model.repo.custom.DepositLocationRepoCustom;

import edu.tamu.weaver.data.model.repo.WeaverOrderedRepo;

public interface DepositLocationRepo extends WeaverOrderedRepo<DepositLocation>, DepositLocationRepoCustom {

    public DepositLocation findByName(String name);

}

File: src/main/webapp/WEB-INF/app/repo/depositLocationRepo.js
vireo.repo("DepositLocationRepo", function (api, DepositLocation, WsApi) {

	var DepositLocationRepo = function() {
		var depositLocationRepo = this;
		depositLocationRepo.model = DepositLocation;
		depositLocationRepo.mapping = api.DepositLocation;
		return depositLocationRepo;
	};

	return new DepositLocationRepo();
	
});